pkg/ttl/ttlworker/session.go

```diff
@@ -87,7 +87,7 @@ func getSession(pool util.SessionPool) (session.Session, error) {
 	originalIsolationReadEngines, restoreIsolationReadEngines := "", false
 
 	se := session.NewSession(sctx, exec, func(se session.Session) {
-		_, err = se.ExecuteSQL(context.Background(), fmt.Sprintf("set tidb_retry_limit=%d", originalRetryLimit))
+		_, err := se.ExecuteSQL(context.Background(), fmt.Sprintf("set tidb_retry_limit=%d", originalRetryLimit))
```

Comment:
It's better to schedule a new PR to fix it. Though it rarely happens, but I think we should cherry-pick it to the previous versions to avoid potential panics here.

Reply:
Yeah! I'll submit a new PR to fix the two bugs (and cherry-pick them back to release version), and leave this PR only related to test.